app/assets/javascripts/plantings.js.coffee

```diff
@@ -3,4 +3,4 @@
 # You can use CoffeeScript in this file: http://jashkenas.github.com/coffee-script/
 
 jQuery ->
-  $('.add-datepicker').datepicker('format' : 'yyyy-mm-dd')
+  $('.add-datepicker').datepicker('format' : 'yyyy-mm-dd')
```

Comment:
This appears to have lost the final blank line which Github doesn't like. Very minor, but if you're making any other changes, it would be good to fix.